core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -531,5 +531,457 @@
     </li>
   </ul>
 </div>
+<style>
```

Comment:
Please add these styles in the corresponding `.css` file. Also regenerate the `.rtl.css` file(s) using `python -m scripts.rtl_css --mode generate`, once you've made changes to the `.css` file(s)

Reply:
Sir, I have added the changes to the "top-navigation-bar.component.rtl.css" file as you have mentioned above but I am unable to regenerate the rtl.css file using  "python -m scripts.rtl_css --mode generate" command, it is given an error " b'/usr/bin/env: \xe2\x80\x98node\xe2\x80\x99: No such file or directory\n". 
Although the website is working absolutely fine from my side.

Reply:
@anuragc10 you have to add the changes in the `top-navigation-bar.component.css` file (and remove the style tag from the .html file) and auto-generate the `top-navigation-bar.component.rtl.css` using the command. I'm not sure why the command isn't working for you, could you please attach a screenshot? Thanks!

Reply:
@Lawful2002  please check, I have removed the styles from the HTML file and added changes in "top-navigation-bar.component.css".

screenshot of the error: 
![image](https://user-images.githubusercontent.com/63997049/156567293-d35660d4-91a4-474c-b227-56c51798e095.png)

Reply:
@Rijuta-s, The terminal is showing the same error.

Reply:
@Rijuta-s, It is giving the same error. I have tried all the steps mentioned above as well as the instructions provided in the documentations.

Reply:
Hi @anuragc10, I have regenerated the .rtl.css files for you (i've created a PR on your repo). Also can you please file a bug for the issue you are facing running the command?
Thanks!

Reply:
Thank You, @Lawful2002  Sir, I have merged the change and will file the issue.